src/ng/plugins/edit-form/edit.form.html

```diff
@@ -1,4 +1,4 @@
-<q-grid:popup id="q-grid-edit-form-popup">
+<q-grid:popup id="q-grid-edit-form-popup-{{$editForm.key}}">
```

Comment:
why do we need that? only one form can be opened per grid on the screen, we can use grid id instead?